NashTeamAlpha/BangazonWeb
# created Product model

## Pull Request Description

Include the applicable information from the list below here 
1. Touched Product.cs model
2. Added columns to table.
## Steps to test
1. switch to branch productModel
2. look in Models folder
3. open Product.cs
4. double check to make sure all properties listed in the issue ticked are implemented correctly.
## Link to Feature Ticket

Provide the issue number that this pull request fixes
#6
